zabbix-probes/zabbix-probes-common/src/main/java/com/indigo/zabbix/utils/CmdbFeignClient.java

```diff
@@ -0,0 +1,30 @@
+package com.indigo.zabbix.utils;
+
+import com.google.gson.JsonElement;
+import com.indigo.zabbix.utils.beans.CmdbResponse;
+import com.indigo.zabbix.utils.beans.ServiceType;
+
+import feign.Param;
+import feign.RequestLine;
+
+/**
+ * Created by jose on 8/02/17.
+ */
+public interface CmdbFeignClient {
+
+  @RequestLine("GET /service/list?include_docs=true")
+  CmdbResponse<ServiceType> services();
+
+  @RequestLine("GET /provider/id/{providerId}/has_many/services?include_docs=true")
+  JsonElement providerInfo(@Param("providerId") String providerId);
+  
+  @RequestLine("GET /provider/list")
+  JsonElement providerList();
+  
+  @RequestLine("GET /image/list")
+  JsonElement providerImages();
+  
+  @RequestLine("GET /provider/id/{providerId}/has_many/services?include_docs=true")

```

Comment:
This has to be the only file in probes-commons. The change was made. Thanks for realizing the mistake